gateway/api_definition_test.go

```diff
@@ -1032,7 +1029,7 @@ func (ts *Test) testPrepareDefaultVersion() (string, *APISpec) {
 func TestGetVersionFromRequest(t *testing.T) {
 
 	versionInfo := apidef.VersionInfo{}
-	versionInfo.Paths.WhiteList = []string{"/foo"}
+	versionInfo.Paths.WhiteList = []string{"/foo", "/v3/foo"}
```

Comment:
is this added to cover the cases in `matchesPath`?

Reply:
Covers some tests cases around L1101, strip version paths